rootfs/etc/profile.d/atmos.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+function _configure_atmos_base_path() {
+	# Leave $ATMOS_BASE_PATH alone if it is already set
+	if [[ -n $ATMOS_BASE_PATH ]]; then
+		if [[ $SHLVL == 1 ]]; then
+			green "# Using configured $ATMOS_BASE_PATH of \"$ATMOS_BASE_PATH\""
+		fi
+		return
+	fi
+
+	# If $GEODESIC_WORKDIR contains both a "stacks" and "components" directory,
+	# use it as the $ATMOS_BASE_PATH
+	if [[ -d "${GEODESIC_WORKDIR}/stacks" ]] && [[ -d "${GEODESIC_WORKDIR}/components" ]]; then
+		export ATMOS_BASE_PATH="${GEODESIC_WORKDIR}"
+		green "# Setting ATMOS_BASE_PATH to \"$ATMOS_BASE_PATH\" based on children of workdir"
```

Comment:
"Children of workdir" is a little vague outside of the context, especially to a Geodesic user who isn't reading the source of this script.

Here's a quick suggestion, but maybe word it differently.

```suggestion
		green "# Atmos base path automatically determined to be current workdir; setting ATMOS_BASE_PATH to \"$ATMOS_BASE_PATH\""
```

Reply:
What is printed out is in a form and style consistent with the other startup output from Geodesic:
```
# Mounting /Users/user into container
# Starting new geodesic session from cloudposse/geodesic:debian
# Exposing port 32380
# Setting ATMOS_BASE_PATH to "/localhost/source/geodesic" based on children of workdir
```

The fact that "children of workdir" is a bit cryptic is OK with me. The purpose of the message is not to _explain_ to the user how the setting was derived, as that just takes too much verbiage for a startup message. The point is to _distinguish which criteria were used_ in setting `ATMOS_BASE_PATH` so if there is a problem, we can more easily diagnose the cause.